bee-crypto/benches/batched_hash.rs

```diff
@@ -23,7 +23,43 @@ fn batched_hasher(input: TritBuf) {
     for _ in hasher.hash_batched() {}
 }
 
-fn unbatched_hasher(input: TritBuf) {
+fn unbatched_hasher_t5b1(input: &TritBuf<T5B1Buf>) {
+    let mut hasher = BatchHasher::new(input.len(), CurlPRounds::Rounds81);
```

Comment:
I'm confused, why is it called unbatched if it batches ?

Reply:
because the batchhasher has both hashers inside it. So if you call hash_unbatched it uses `CurlP`, and if you call hash_batched it uses `BatchedCurlP`.

Reply:
Eww don't mind me, I'm tired, didn't see the last line